packages/instrumenter/src/transformers/babel-transformer.ts

```diff
@@ -139,6 +197,7 @@ export const transformBabel: AstTransformer<ScriptFormat> = (
    */
   function collectMutants(path: NodePath) {
     return [...mutate(path)]
+      .filter((mutant) => !isMutantInDisabledMap(path.node.loc, mutant))
```

Comment:
I don't want the mutants to be _filtered out_, instead I want them to be collected, but with an `ignoreReason` set. That way, they will still end up in the report, but won't be placed, tested or influence the mutation score.